src/components/FieldLevelHelp/FieldLevelHelp.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../Icon';
+import { Popover } from '../Popover';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+
+/**
+ * FieldLevelHelp Component for Patternfly React
+ */
+const FieldLevelHelp = ({ children, contentType, content, ...props }) => {
+  const trigger = contentType === 'popover' ? 'click' : 'hover focus';
+  const htmlContent = (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: content
+      }}
+    />
+  );
+  const overlay =
+    contentType === 'popover' ? (
+      <Popover id="{contentType}">{htmlContent}</Popover>
+    ) : (
+      <Tooltip id="{contentType}">{htmlContent}</Tooltip>
+    );
+  const rootClose = true;
+
+  return (
+    <div>
+      <label>{children + ' '}</label>
+      <OverlayTrigger
+        overlay={overlay}
+        placement={'top'}
+        trigger={trigger.split(' ')}
+        rootClose={rootClose}
+      >
+        <Icon
+          className="fa-fw"
+          type="pf"
+          name={'info'}
+          style={{ color: '#0088ce' }}
+        />
+      </OverlayTrigger>
+    </div>
+  );
+};
+
+FieldLevelHelp.propTypes = {
+  /** additional fieldlevelhelp classes */
+  /** Content type: popover or tooltip  */
+  contentType: PropTypes.string,
+  /** Contents displayed with popover or tooltip  */
+  content: PropTypes.string,
+  /** children nodes  */
+  children: PropTypes.node
```

Comment:
does it make sense to have a props to force the popover to be open? (instead on just on hover/click)

Reply:
Added a new prop - `close`, to close/open the popover if the use case needs it.
(if popover is forced to be open, it can be closed after clicking on the 'i' icon)

0db70f4